### DISCUSSION TITLE
get status of devices

### DISCUSSION THREAD

**Discussion by kitebuggylux** [pzim-devdata/tahoma#10] [Ideas]
Hi.

It will be nice to integrate the possibility to get values of the status of devices i.a.: closed 25% or so.

Thanx

Al

**Comment by pzim-devdata** [comment]
Hello,
Yes you can know if it's open or closed with the `sensor` category:
`tahoma --password 'secret ' get sensor ["Kummer Alex"] --remote`

```
$ tahoma -la

List of actions by categories :
For the category SHUTTER : [open,close,stop,my,NUMBER]
For the category SPOTALARM : [on,off,toggle]
For the category PLUG : [on,off,toggle]
For the category LIGHT : [on,off,toggle]
For the category ALARM : [arm,disarm,partial,arm_night,arm_away]
For the category HEATER : [comfort,comfort-1,comfort-2,eco,frostprotection,off,standby,manual,auto,prog,NUMBER]
For the category SUNSCREEN : [open,close,stop,my,NUMBER]
For the category SCENE : [on,activate,launch,execute]
For the category SENSOR : [get,get_state,get_position,get_lumens,get_temperature]
```


If you want to know the state of all the sensors with the `kummer` name inside you can use this command :
`tahoma --password 'secret ' get sensor Kummer --remote`

**Reply by pzim-devdata** [reply]
Unfortunately, it's impossible to know the exact position of a shutter (only if it's open or closed).
